ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -957,11 +963,28 @@ function scatterProcessData(
             processedData.push({ x, y });
             processedXData.push(x);
             processedYData.push(y);
+            if (xRangeNeeded) {
+                xDataMax = Math.max(xDataMax, x);
+                xDataMin = Math.min(xDataMin, x);
+            }
+            if (yRangeNeeded) {
+                yDataMax = Math.max(yDataMax, y);
+                yDataMin = Math.min(yDataMin, y);
+            }
         } else {
             cropped = true;
         }
     }
 
+    if (xRangeNeeded) {
+        xAxis.options.max = xAxis.options.max || xDataMax;
+        xAxis.options.max = xAxis.options.min || xDataMin;
```

Comment:
I think this fails if `options.min` or `max` is 0. So we want a nullish test:

`xAxis.options.max ??= xDataMax;`